README.md

```diff
@@ -50,6 +50,11 @@ source env/bin/activate
 pip install -r requirements.txt
 ```
 
+If you want to evalute on NVIDIA GPU, pip install `flash-attn` as your requirements.
+```bash
+pip install flash-attn
+```
+
```

Comment:
Suggest changing this section to:

For evaluating on NVIDIA GPUs, please install `flash-attn` by referring to the [flash attention repo](https://github.com/Dao-AILab/flash-attention).